### PR TITLE
Added react-native-toast-message package for displaying toast messages

### PR DESCRIPTION
**Pull request by vinayak-trivedi**
Solves issue #115. Image for reference -> 
Screenshot_1656736324

### CODE REVIEW

File: src/helpers/LaunchCamera.tsx [hunk 1]
```diff
@@ -42,7 +43,13 @@ const takePicture = async (
           setResponse,
         );
       } else {
-        Alert.alert('Camera permission denied');
+        Toast.show({
+          type: 'error',
+          text1: 'Permission denied',
```
Comment (shreya-mishra) [inline src/helpers/LaunchCamera.tsx]: 1. give text1 and text2 to some meaningful name.
2. Move 'Permission denied' to en.ts file.

File: src/helpers/LaunchCamera.tsx [hunk 2]
```diff
@@ -42,7 +43,13 @@ const takePicture = async (
...
+          text2: 'Camera Permission denied',
```
Comment (shreya-mishra) [inline src/helpers/LaunchCamera.tsx]: Move 'Camera Permission denied' to en.ts file and then use the string over here

File: src/screens/HomeScreen/HomeScreen.tsx [hunk 3]
```diff
@@ -29,7 +30,15 @@ const HomeScreen = () => {
             }),
           );
         })
-        .catch((err) => Alert.alert(err))
+        .catch((err) => {
+          Toast.show({
+            type: 'error',
+            text1: 'Error',
```
Comment (shreya-mishra) [inline src/screens/HomeScreen/HomeScreen.tsx:36]: same as above

File: src/screens/HomeScreen/HomeScreen.tsx [hunk 4]
```diff
@@ -29,7 +30,15 @@ const HomeScreen = () => {
...
+            text2: err,
```
Comment (shreya-mishra) [inline src/screens/HomeScreen/HomeScreen.tsx:37]: same as above

**Review by shreya-mishra — COMMENTED** [review]
Else everything looks good. The package is also lightweight which is nice.

File: src/helpers/LaunchCamera.tsx (hunk 1 above)
Comment (vinayak-trivedi) [reply src/helpers/LaunchCamera.tsx]: text 1 text2 are properties provided by the package only we can't change that

File: src/helpers/LaunchCamera.tsx (hunk 2 above)
Comment (vinayak-trivedi) [reply src/helpers/LaunchCamera.tsx]: I will do it

File: src/helpers/LaunchCamera.tsx (hunk 1 above)
Comment (shreya-mishra) [reply src/helpers/LaunchCamera.tsx]: aah ok my bad

**Review by shreya-mishra — APPROVED** [review]

**Review by shreya-mishra — APPROVED** [review]
LGTM

**Pull request merged**